.github/workflows/haskell.yml

```diff
@@ -0,0 +1,12 @@
+name: Haskell CI
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: cachix/install-nix-action@v6
+    - uses: cachix/cachix-action@v2
+      with:
+        name: kutyel
```

Comment:
Does this mean we're publishing to cachix under `kutyel`?

Reply:
I'm afraid yes, should I create a 47deg cachix account @juanpedromoreno? 😉